package-testing/sdk-test-runner/test-sdk.sh

```diff
@@ -67,6 +70,7 @@ export SDK_RELAY_PORT="${SDK_RELAY_PORT:-4000}"
 export EPPO_SCENARIO_FILE="${EPPO_SCENARIO_FILE:-scenarios.json}"
 export EPPO_TEST_DATA_PATH="${EPPO_TEST_DATA_PATH:-./test-data}"
 
+PLATFORM="${PLATFORM:-linux}"
```

Comment:
Did you consider no default? I know it is tedious to add but could add clarity to where it is run

Reply:
Ya, it would be best to be explicit here.